src/repositories/baseRepository.ts

```diff
@@ -73,7 +73,7 @@ export abstract class BaseRepository {
 
   protected async findOne(query: any, errorMsg: string, options: mongodb.FindOptions = {}): Promise<any> {
     try {
-      return this.promiseTimeoutS(
+      return await this.promiseTimeoutS(
```

Comment:
Thanks for fixing! Super weird that the lack of `await` ends up breaking the test here! 😮 I would expect that this `await` is unnecessary since the function already returns a promise.

Reply:
Agree. I was a bit shocked myself, but I guess the `findOne` would return an empty value before the actual `this.promiseTimeoutS` finished because it wasn't awaiting.